Algorithm-Roman-Numeral-Converter.md

```diff
@@ -6,6 +6,13 @@
 
 You will create a program that converts an integer to a Roman Numeral.
 
+#### Relevant Links
+
+- [Roman Numerals](http://www.mathsisfun.com/roman-numerals.html)
+- [Array.splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)
```

Comment:
Change this so it links to our internal article: [`JS-Array-Prototype-Splice`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Splice.md).